### PR TITLE
quack 0.48 (new formula)

### PR DESCRIPTION
**Pull request by alehed**
The compilation gives some warnings, but I think they can safely be ignored.

### CODE REVIEW

File: Formula/quack.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require File.expand_path("../../Homebrew/emacs_formula", __FILE__)
+
+class Quack < EmacsFormula
+  desc "Enhanced Emacs support for Racket and Scheme"
+  homepage "http://www.neilvandyke.org/quack/"
+  url "http://www.neilvandyke.org/quack/quack.el"
```
Comment (dunn) [inline Formula/quack.rb]: We'll need a versioned URL so that the formula doesn't break when it's updated.

File: Formula/quack.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  sha256 "5d6f103307c9f3809f3fa9a7179a08c05add3afdf134553eae8ae19dcfccce5b"
+  version "0.48"
+
```
Comment (dunn) [inline Formula/quack.rb]: There's some trailing whitespace here, could you remove it?

**Comment by dunn** [comment]
Thanks! Just a few comments.

File: Formula/quack.rb (hunk 1 above)
Comment (alehed) [inline Formula/quack.rb]: Unfortunately, I don't think there is one right now. Do I have to create a mirror on github?

Comment (dunn) [inline Formula/quack.rb]: Could you email the developer and see if they'll create one? Thanks!

Comment (alehed) [inline Formula/quack.rb]: I'll try that.

**Comment by dunn** [comment]
Feel free to open a new PR when there's a stable release!

**Comment by alehed** [comment]
Sure, I will do that. I wrote the maintainer an email, but haven't received an answer yet. The updates seem to be bugfixes-only so I guess it's not such a high priority for the maintainer.

**Pull request closed without merging**